src/Components/styles/ProfilePage.css

```diff
@@ -0,0 +1,51 @@
+p, h4, h2{
+    text-align: center;
+}
+
+h2 {
+    font-weight: 500;
+    font-size: 48px;
```

Comment:
Question @zhengwin . Are font sizes dynamic (or independent of the pixel density of the screen)?? We should have a common set of values for font sizes to match the designs rather than hardcode values everywhere (title, body text, heading text etc). Will be hella pain to maintain if we need to change.

Reply:
All font-sizes should be defined with rem instead of px. 

I agree that we should have a common set of values for font sizes, we should have a meeting next week to talk about it